scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -700,14 +700,14 @@ s32 pick_idle_cpu(struct task_struct *p, s32 prev_cpu,
 		return -1;
 	}
 
-	pref_idle_cpumask = bpf_cpumask_create();
-
 	if (layer->idle_smt) {
 		idle_cpumask = scx_bpf_get_idle_smtmask();
 	} else {
 		idle_cpumask = scx_bpf_get_idle_cpumask();
 	}
 
+	pref_idle_cpumask = bpf_cpumask_create();
```

Comment:
Moving this here because on older kernels the verifier loses the bpf_cpumask tracking easily.